src/Sylius/Behat/Page/Admin/Product/ShowPageInterface.php

```diff
@@ -14,10 +14,13 @@
 namespace Sylius\Behat\Page\Admin\Product;
 
 use FriendsOfBehat\PageObjectExtension\Page\SymfonyPageInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
 
 interface ShowPageInterface extends SymfonyPageInterface
 {
     public function getName(): string;
 
     public function isSimpleProductPage(): bool;
+
+    public function specifyChannel(ChannelInterface $channel): void;
```

Comment:
`showProductInChannel`

Reply:
And do we need the whole channel there? Maybe just a channel name would be enough?